analysis/management/commands/multi_character_analysis.py

```diff
@@ -0,0 +1,80 @@
+import os
+from collections import Counter
+from string import ascii_letters, digits, punctuation, whitespace
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models.functions import Length
+from lxml import html
+from nltk.tokenize import stanford_segmenter
+
+from analysis.models import SingleCount, MultiCount
+from dictionary.models import Entry
+
+
+class Command(BaseCommand):
+    help = '''
+    parse a list of html documents and bag the occurence of dictionary entries.  
+    '''
+
+    def __init__(self):
+        self.base_dir = os.path.dirname(os.path.abspath(__file__))
+        os.environ['JAVAHOME'] = settings.JAVAHOME
+        os.environ['CLASSPATH'] = settings.CLASSPATH
+        self.segmenter = stanford_segmenter.StanfordSegmenter(
+            path_to_jar=settings.STFD_SEG,
+            path_to_slf4j=settings.SLF4J,
+            path_to_sihan_corpora_dict=settings.SIHAN_DICT,
+            path_to_model=settings.MODEL,
+            path_to_dict=settings.DICT
+            )
```

Comment:
One thing I like to do when I have a bunch of settings that go together like this is make it a `dict` in the settings:

```python
STANFORD_SEGMENTER_OPTS = {
    'path_to_jar': '...',
    'path_to_slf4j': '...',
    ... }
```

Then in your code you just have to do:

```python
self.segmenter = stanford_segmenter.StanfordSegmenter(**settings.STANFORD_SEGMENTER_OPTS)
```